Modules/makesetup

```diff
@@ -98,11 +98,23 @@ CYGWIN*) if test $libdir = .
 esac
 
 # Main loop
-for i in ${*-Setup}
+for setupfile in ${*-Setup}
 do
-	case $i in
+	# Check if Setup file does not exist.  In that case, use
+	# Setup.dist from libdir (source) folder.  In the normal case,
+	# the user does not create or modify Setup and so Setup.dist
+	# from the source distribution is used as-is.
+	case $setupfile in
+	*/Setup)
```

Comment:
This pattern matching on */Setup is a bit ugly.  Because makesetup can be called with different arguments (not just from 'configure'), we have to handle running on different Setup files.  Eons ago, makesetup + Setup would be used instead of setup.py for 3rd party extension modules.  Now, I suspect 'makesetup' might only be used from within the cpython build.